josemak25/hairx-client
# Ft hair status dialogue

#### What does this PR do?

Show a modal that shows duration of colored hair

#### Description of Task to be completed?

Have the screen show the following:

1. a modal.
2. Duration options

#### How should this be manually tested?

After cloning the repo,
![IMG_2713](https://user-images.githubusercontent.com/51112934/80926098-d6fc0580-8d8c-11ea-9ef1-95d9e9fccac4.PNG)

`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots (if appropriate)
